src/main/java/frc/robot/Robot.java

```diff
@@ -34,6 +34,7 @@ public void robotPeriodic() {
         // and running subsystem periodic() methods. This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
         CommandScheduler.getInstance().run();
+        robotContainer.periodic();
```

Comment:
for practice, let's make sure that this only runs in teleop. We don't want states to be overwritten during auto